crocket/src/main/java/com/crocket/model/DrawableEntity.java

```diff
@@ -35,9 +35,17 @@ public int getHeight() {
         return height;
     }
 
-    public int getRotation() {
+    public int getRotation(){
```

Comment:
These should have a space before `{` to be consistent with the rest of the project